hyperapp.d.ts

```diff
@@ -17,10 +17,19 @@ export interface VNode<Attributes = {}> {
  *
  * @memberOf [VDOM]
  */
-export interface Component<Attributes = {}, State = {}, Actions = {}> {
-  (attributes: Attributes, children: Array<VNode | string>):
-    | VNode<Attributes>
```

Comment:
Initially I found the initial  '|' confusing. I did not know you could do this and looked for a preceding token. It does give better layout though so I'll leave it as a stylistic issue and not say anything else :)

Reply:
This has to do with the max-width of 80 columns i guess...

Reply:
it's more the leading | than the wrapping ie
```
| VNode<Attributes> | View<State, Actions>
```
rather than 
```
VNode<Attributes> | View<State, Actions>

```

Reply:
It's the language. So get used to Typescript

Reply:
like I said - it's stylistic so not part of the PR